src/utils/deepObject.js

```diff
@@ -0,0 +1,91 @@
+const _ = require('lodash');
+const logger = require('../logger');
+
+const validateKeys = (newKeys) => newKeys.reduce((validatedKeys, key) => {
+	if (key && key !== '') {	// remove invalid pathes
+		validatedKeys.push(key);
+	}
+	return validatedKeys;
+}, []);
+
+/**
+ * @param  {...String||Array} paths 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4646529)